entities/author.rb

```diff
@@ -0,0 +1,12 @@
+class Author
+  attr_accessor :name, :bio
+
+  def initialize(name, bio)
+    @name = name
```

Comment:
Let's add validation. It should be a String and it shouldn't be empty
Please add validations to other entities as well